dev/com.ibm.ws.jdbc/src/com/ibm/ws/rsadapter/jdbc/WSJdbcProxyMethod.java

```diff
@@ -211,9 +211,10 @@
         //  oracle.jdbc.internal.OracleConnection.unwrap()
         
         //// PostgreSQL methods
-        unsafeMethods.add("getLargeObjectAPI"); // org.postgresql.PGConnection.getLargeObjectAPI() -- LargeObject API is not allowed because it has operations that can commit transactions on the underlying PG connection
-        unsafeMethods.add("getFastpathAPI"); // org.postgresql.PGConnection.getFastpathAPI() -- this is deprecated anyway on the PG API
```

Comment:
was fastpath blocked because it was deprecated or unsafe?  If the latter, how did we address its safety?

Reply:
it was previously blocked because some code paths can perform queries on the underlying connection. After discussing that setAutosave, my understanding on what is safe vs. unsafe has relaxed a bit, and this operation seems fine to permit.